webdriver/tests/bidi/input/perform_actions/pointer_touch.py

```diff
@@ -158,8 +158,8 @@ async def test_touch_pointer_properties_tilt_twist(
             width=23,
             height=31,
             pressure=0.78,
-            tilt_x=21,
-            tilt_y=-8,
+            altitude_angle=1.1827416313714896,
```

Comment:
looks like the test names and the comments have to be updated? the comment says "This test only covers the tilt/twist properties"